app/controllers/issue_monitoring_reviews_controller.rb

```diff
@@ -19,6 +19,15 @@ def create
     end
   end
 
+  def unlink
```

Comment:
reminder for self: @kbighorse points out this ought to be `destroy` rather than `unlink`